modules/HapticGlove_module/src/GloveControlHelper.cpp

```diff
@@ -211,13 +218,19 @@ bool GloveControlHelper::stopPalmVibrotactileFeedback()
 bool GloveControlHelper::stopVibrotactileFeedback()
 {
     std::fill(m_desiredVibrotactileValues.begin(), m_desiredVibrotactileValues.end(), 0.0);
-    return m_pImp->setFingertipVibrotactileValues(m_desiredVibrotactileValues);
+    return true;
 }
 
 bool GloveControlHelper::stopForceFeedback()
 {
     std::fill(m_desiredForceValues.begin(), m_desiredForceValues.end(), 0.0);
-    return m_pImp->setFingertipForceFeedbackValues(m_desiredForceValues);
+    return true;
+}
+
+bool GloveControlHelper::stopAllHapticFeedback()
+{
+    std::fill(m_desiredHapticValues.begin(), m_desiredHapticValues.end(), 0.0);
+    return m_pImp->setFingertipHapticFeedbackValues(m_desiredForceValues, m_desiredVibrotactileValues);
```

Comment:
Why setting the ``m_desiredHapticValues`` to zero if then you are passing ``m_desiredForceValues`` and ``m_desiredVibrotactileValues`` to the method?

In this sense, is ``m_desiredHapticValues`` needed at all?

Reply:
Now that I look at it again, actually, there is no need for the `stopAllHapticFeedback()` method. Removing it.